Sources/PklSwift/Utils.swift

```diff
@@ -85,3 +85,12 @@ public func resolvePaths(_ paths: String...) -> String {
 }
 
 public let absoluteUriRegex = try! Regex("\\w+:")
+
+public func tempDir() throws -> String {
+    (try FileManager.default.url(for: .itemReplacementDirectory, in: .userDomainMask, appropriateFor: URL(filePath: "/")!, create: true)).absoluteString
```

Comment:
I'd be inclined to let `tempDir()` return a `URL`, considering all the uses, but it's non-blocking.

Reply:
Updated!